tasks/2-build-tag-push-ace-bar.yaml

```diff
@@ -0,0 +1,139 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: ibm-build-tag-push-ace-bar
+  annotations:
+    description: Executes logic to build, tag and push a container image using the intermediate sha tag to the image-url
+    app.openshift.io/description: Executes logic to build, tag and push a container image using the intermediate sha tag to the image-url
+    app.openshift.io/vcs-uri: https://github.com/IBM/ibm-garage-tekton-tasks
+    app.openshift.io/vcs-ref: master
+  labels:
+    version: 0.0.0
+spec:
+  params:
+  - name: git-url
+  - name: git-revision
+    default: master
+  - name: source-dir
+    default: /source
+  - name: image-server
+    default: ""
+  - name: image-namespace
+    default: ""
+  - name: image-repository
+    default: ""
+  - name: image-tag
+    default: ""
+  - name: BUILDER_IMAGE
+    default: quay.io/buildah/stable:v1.15.0
+  - name: DOCKERFILE
+    default: ./Dockerfile
+  - name: CONTEXT
+    default: .
+  - name: TLSVERIFY
+    default: "false"
+  - name: FORMAT
+    default: docker
+  - name: STORAGE_DRIVER
+    description: Set buildah storage driver
+    default: overlay
+  - name: app-name
+    default: ""
+  - name: ace-project
+    default: ""
+  volumes:
+  - name: varlibcontainers
+    emptyDir: {}
+  - name: source
+    emptyDir: {}
+  stepTemplate:
+    volumeMounts:
+    - name: source
+      mountPath: $(params.source-dir)
+  steps:
+  - name: git-clone
+    image: quay.io/ibmgaragecloud/alpine-git
+    env:
+    - name: GIT_PASSWORD
+      valueFrom:
+        secretKeyRef:
+          key: password
+          name: git-credentials
+          optional: true
+    - name: GIT_USERNAME
+      valueFrom:
+        secretKeyRef:
+          key: username
+          name: git-credentials
+          optional: true
+    script: |
+      set +x
+      if [[ -n "${GIT_USERNAME}" ]] && [[ -n "${GIT_PASSWORD}" ]]; then
+          git clone "$(echo $(params.git-url) | awk -F '://' '{print $1}')://${GIT_USERNAME}:${GIT_PASSWORD}@$(echo $(params.git-url) | awk -F '://' '{print $2}')" $(params.source-dir)
+      else
+          set -x
+          git clone $(params.git-url) $(params.source-dir)
+      fi
+      set -x
+      cd $(params.source-dir)
+      git checkout $(params.git-revision)
+  - name: compile
+    image: docker.io/rsundara/ace-build
```

Comment:
Open an issue to track the work on trying to get the source Dockerfile for this image

Reply:
https://github.com/ibm-garage-cloud/planning/issues/787 has been opened.